pkg/cmd/experimental/bundlesecret/bundle-secret.go

```diff
@@ -124,7 +124,7 @@ func (o *CreateSecretOptions) CreateSecret() (*kapi.Secret, error) {
 
 			for _, item := range fileList {
 				itemPath := path.Join(source, item.Name())
-				if item.IsDir() {
+				if item.IsDir() || (!item.Mode().IsRegular()) {
```

Comment:
So from the go docs, it looks like item.Mode.IsRegular will return false for a directory as well.

Can you structure this:

```shell
if !item.Mode().IsRegular() {
   if o.Stderr != nil && o.Quiet != true {
     fmt.Fprintf(o.Stderr, "Skipping resource %s\n", itemPath)
   }
}
```